### PR TITLE
adding stopwatch-timer application.

### PR DESCRIPTION
**Pull request by someshfengde**
I've updated the code and now stopwatch-timer is working perfectly fine 😄 

Thanks, @Coding-with-Adam  for guiding me on this one. 

Now the application is working without errors. 

### This post also helped me for solving this issue. 
https://community.plotly.com/t/duplicate-callback-outputs-solution-api-discussion/55909/2

### CODE REVIEW

File: pages/stopwatch-timer.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import dash
+
+from utils.code_and_show import example_app
+
+
+dash.register_page(__name__, description="stopwatch timer application with 2 buttons ")
+
+
+def layout():
+    return example_app(f"pages/examples/stopwatch.py")
```
Comment (Coding-with-Adam) [inline pages/stopwatch-timer.py:10]:
```suggestion
dash.register_page(
    __name__,
    description = "Stopwatch timer application with 1 button and an input field.",
    layout_type = "general",
    components_type = ["leddisplay", "input", "interval", "powerbutton"],
    graph_type = "none", 
    callback_type = "general"
)

filename = __name__.split("pages.")[1]

# make a mini layout with the links to the docs that would go under each app
# all_links: {"input": "https://dash.plotly.com/dash-core-components/input",
# "interval": "https://dash.plotly.com/dash-core-components/interval",
# "dash-daq": "https://dash.plotly.com/dash-daq"} 
# links = html.Div([all_links])


def layout():
    return example_app(f"pages/examples/{filename}.py")
```

**Review by Coding-with-Adam — CHANGES_REQUESTED** [review]
Hi @someshfengde 
Thank you for making updates to the code. 
Can you please update the png image of the app with a new image that pertains to the new code. And can you please accept all the suggested changes I added.

File: pages/examples/stopwatch.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+])
+
+
+@app.callback(
+    Output('stopwatch-y', 'value'),
+    Input('trigger-while-button-on', 'n_intervals'),
+    Input('stopwatch-input', 'value'),
+    Input("stopwatch-start", 'on'),
+    Input('stopwatch-y', 'value'),
+)
```
Comment (Coding-with-Adam) [inline pages/examples/stopwatch.py:48]:
```suggestion
@app.callback(
    Output('stopwatch-x-y', 'value'),
    Input('stopwatch-x-trigger-while-button-on', 'n_intervals'),
    Input('stopwatch-x-myinput', 'value'),
    Input("stopwatch-x-start", 'on'),
    Input('stopwatch-x-y', 'value'),
)
```

File: pages/examples/stopwatch.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            placeholder="enter number of seconds here",
+            )
+        ]
+        ),
+   
+    dcc.Interval(id='trigger-while-button-on',
+                 interval=1*1000,
+                 n_intervals=0),
+    daq.PowerButton(
+        id='stopwatch-start',
```
Comment (Coding-with-Adam) [inline pages/examples/stopwatch.py:33]:
```suggestion
        id='stopwatch-x-start',
```

File: pages/examples/stopwatch.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+        id='stopwatch-y',
+        label="Stopwatch",
+        value=6
+    ),
+ 
+    html.Center( 
+        [
+            html.Header("Enter seconds"),
+            dcc.Input(
+            id="stopwatch-input",
```
Comment (Coding-with-Adam) [inline pages/examples/stopwatch.py:22]:
```suggestion
            id="stopwatch-x-myinput",
```

File: pages/examples/stopwatch.py [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+import dash_core_components as dcc
+import dash_html_components as html
+import time 
+
+app = dash.Dash(__name__)
+
+app.layout = html.Div([
+    
+    daq.LEDDisplay(
+        id='stopwatch-y',
```
Comment (Coding-with-Adam) [inline pages/examples/stopwatch.py:13]:
```suggestion
        id='stopwatch-x-y',
```

File: pages/examples/stopwatch.py [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+            html.Header("Enter seconds"),
+            dcc.Input(
+            id="stopwatch-input",
+            type="number",
+            placeholder="enter number of seconds here",
+            )
+        ]
+        ),
+   
+    dcc.Interval(id='trigger-while-button-on',
```
Comment (Coding-with-Adam) [inline pages/examples/stopwatch.py:29]:
```suggestion
    dcc.Interval(id='stopwatch-x-trigger-while-button-on',
```

**Review by someshfengde — COMMENTED** [review]
adding changes

**Review by Coding-with-Adam — APPROVED** [review]

**Pull request merged**